.github/workflows/docs.yml

```
@@ -0,0 +1,34 @@
+name: Docs Pipeline
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)